packages/xchain-bitcoin/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # v.x.x.x (2020-XX-XX)
 
+# v.0.6.0 (2020-12-11)
+
 ### Update
 
 - Update dependencies
```

Comment:
In addition ^:

Add
- `createTxInfo` to support transactions using Ledger
- `getDerivePath` helper

Breaking changes:
- Remove deprecated stuff of `electrs`
- Extract constants to `src/const`
- Extract common types to `types/common`

Fix: 
- Update exports